clang/test/Driver/linker-wrapper-sycl-win.cpp

```diff
@@ -0,0 +1,12 @@
+// REQUIRES: system-windows
+
+/// Check for list of commands for standalone clang-linker-wrapper run for sycl
+// RUN: clang-linker-wrapper -sycl-device-library-location=%S/Inputs -sycl-device-libraries=libsycl-crt.new.obj,libsycl-complex.new.obj -sycl-post-link-options="SYCL_POST_LINK_OPTIONS" -llvm-spirv-options="LLVM_SPIRV_OPTIONS" "--host-triple=x86_64-unknown-linux-gnu" "--triple=spir64" "--linker-path=/usr/bin/ld" "--" HOST_LINKER_FLAGS "-dynamic-linker" HOST_DYN_LIB "-o" "a.out" HOST_LIB_PATH HOST_STAT_LIB %S/Inputs/test-sycl.o --dry-run 2>&1 | FileCheck -check-prefix=CHK-CMDS %s
```

Comment:
I'm confused about the `--host-triple` value here as this test is Windows specific.  Shouldn't we be testing `x86_64-pc-windows-msvc` instead?

Reply:
cut-paste issue. Good catch. Thanks